Automattic/jetpack
# Setup blogroll attributes and context

This PR wires the attributes and context that we need to the blogroll block
We have the settings defined here: p9Jlb4-8nF-p2#block-settings

Fixes #
https://github.com/Automattic/wp-calypso/issues/80909

## Proposed changes:
* Add the blogroll context
* Add the blogroll item attributes

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:

* Checkout this branch
* Build jetpack
* Apply this patch [print_attributes.patch](https://github.com/Automattic/jetpack/files/12498972/print_attributes.patch)
* Edit a post and add the blogroll block
* Check the console and the attributes should be printed
* When you change the blogroll settings the changes should also be printed

<img width="839" alt="Screenshot 2023-09-01 at 16 48 40" src="https://github.com/Automattic/jetpack/assets/7000684/170fdc27-c54b-42ef-9a4b-f0cb4d1a4a55">
